docs/resources.md

```diff
@@ -162,6 +162,60 @@ spec:
       value: gcr.io/staging-images/kritis
 ```
 
+#### Surfacing the image digest built in a task
+
+To surface the image digest in the output of the `taskRun` the builder tool should produce this information in a [oci-layout-image](https://github.com/opencontainers/image-spec/blob/master/image-layout.md) `index.json` file. This file should be placed on a location as specified in the task definition under the resource `outputImagePath`
+
+for example this build-push task defines the `outputImagePath` for the `buildImage` resource in `/worksapce/buildImage`
```

Comment:
super nitpick: previous sentence should end with `.`, this word should be capitalized